codec/src/main/java/io/netty/handler/codec/DefaultHeaders.java

```diff
@@ -1143,85 +854,43 @@ public long getTimeMillisAndRemove(T name, long defaultValue) {
     }
 
     @Override
+    @SuppressWarnings("unchecked")
```

Comment:
Actually here we can't have the `@SuppressWarnings("unchecked")` apply to a return statement. We would need a temp variable.  Lets re-address in https://github.com/netty/netty/pull/4038 where most of this class changes anyways.